nima/http2/webserver/src/main/java/io/helidon/nima/http2/webserver/spi/Http2SubProtocolProvider.java

```diff
@@ -16,26 +16,37 @@
 
 package io.helidon.nima.http2.webserver.spi;
 
-import io.helidon.config.Config;
+import io.helidon.nima.webserver.ProtocolConfigs;
+import io.helidon.nima.webserver.spi.ProtocolConfig;
 
 /**
  * {@link java.util.ServiceLoader} provider interface for HTTP/2 sub-protocols.
```

Comment:
Isn't this an implementation detail - I would rather see this on the impl and not the iface contract.

Reply:
I don't understand your comment.
HTTP/2 sub-protocols is an extension point for HTTP/2, used by grpc (right now). This is not an impelmentation details, this is proper SPI. 
Can you please explain?

Reply:
Referring to how it gets loaded (i.e., loaded by ServiceLoader) is an implementation detail and doesn't belong on the contract.

Reply:
The contract is a `ServiceLoader` provider interface, explicitly mentioned to be loaded by service loader in `module-info.java`. This is THE contract of this service.